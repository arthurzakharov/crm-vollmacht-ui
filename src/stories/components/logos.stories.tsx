import { Meta, StoryObj } from "@storybook/react";
import Logos from "@components/logos";
import MaxWidthDecorator from "@stories/decorators/max-width-decorator";

type Story = StoryObj<typeof Logos>;

const meta: Meta<typeof Logos> = {
  title: "COMPONENTS/Logos",
  component: Logos,
  parameters: {
    MaxWidthDecorator: 320,
  },
  decorators: [MaxWidthDecorator],
};

export const TlsAndTuv: Story = {
  name: "TlsAndTuv",
  args: {
    show: ["tls", "tuv"],
  },
};

export const TlsOnly: Story = {
  name: "Tls Only",
  args: {
    show: ["tls"],
  },
};

export const TuvOnly: Story = {
  name: "Tuv Only",
  args: {
    show: ["tuv"],
  },
};

export const NoLogos: Story = {
  name: "No Logos",
  args: {
    show: [],
  },
};

export const DefaultValues: Story = {
  name: "Default Values",
  args: {
    show: undefined,
  },
};

export default meta;
