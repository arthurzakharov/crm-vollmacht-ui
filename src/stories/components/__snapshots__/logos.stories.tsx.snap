// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENTS/Logos DefaultValues smoke-test 1`] = `
<div style="max-width: 320px;">
  <div class="logos">
    <div class="logos__tls">
      <img alt="tls-icon"
           src="/src/assets/png/tls/tls.png"
           class="logos__icon"
      >
      <span class="logos__text">
        Höchste
      </span>
      <span class="logos__text">
        Datensicherheit.
      </span>
    </div>
    <img alt="tuv-icon"
         src="/src/assets/jpeg/tuv/tuv.jpeg"
         class="logos__tuv"
    >
  </div>
</div>
`;

exports[`COMPONENTS/Logos NoLogos smoke-test 1`] = `
<div style="max-width: 320px;">
  <div class="logos">
  </div>
</div>
`;

exports[`COMPONENTS/Logos TlsAndTuv smoke-test 1`] = `
<div style="max-width: 320px;">
  <div class="logos">
    <div class="logos__tls">
      <img alt="tls-icon"
           src="/src/assets/png/tls/tls.png"
           class="logos__icon"
      >
      <span class="logos__text">
        Höchste
      </span>
      <span class="logos__text">
        Datensicherheit.
      </span>
    </div>
    <img alt="tuv-icon"
         src="/src/assets/jpeg/tuv/tuv.jpeg"
         class="logos__tuv"
    >
  </div>
</div>
`;

exports[`COMPONENTS/Logos TlsOnly smoke-test 1`] = `
<div style="max-width: 320px;">
  <div class="logos">
    <div class="logos__tls">
      <img alt="tls-icon"
           src="/src/assets/png/tls/tls.png"
           class="logos__icon"
      >
      <span class="logos__text">
        Höchste
      </span>
      <span class="logos__text">
        Datensicherheit.
      </span>
    </div>
  </div>
</div>
`;

exports[`COMPONENTS/Logos TuvOnly smoke-test 1`] = `
<div style="max-width: 320px;">
  <div class="logos">
    <img alt="tuv-icon"
         src="/src/assets/jpeg/tuv/tuv.jpeg"
         class="logos__tuv"
    >
  </div>
</div>
`;

exports[`COMPONENTS/Logos TuvOnly smoke-test 2`] = `
<div style="max-width: 320px;">
  <div class="logos">
    <img alt="tuv-icon"
         src="/src/assets/jpeg/tuv/tuv.jpeg"
         class="logos__tuv"
    >
  </div>
</div>
`;

exports[`COMPONENTS/Logos TuvOnly smoke-test 3`] = `
<div style="max-width: 320px;">
  <div class="logos">
    <img alt="tuv-icon"
         src="/src/assets/jpeg/tuv/tuv.jpeg"
         class="logos__tuv"
    >
  </div>
</div>
`;
